src/withUser.js

```diff
@@ -8,11 +8,16 @@ export default Wrapped => {
       user: null
     };
 
+    handleError(error) {
```

Comment:
If we write this as an arrow function class property, we don't have to `bind` below:
```js
handleError = error => {
   ...
}
```